### DISCUSSION TITLE
How do I configure this?

### DISCUSSION THREAD

**Discussion by kaihendry** [robbrad/UKBinCollectionData#1054] [Q&A]
Is there a URL to configure the service? I don't see an option to configure the selenium driver. Utterly confused.

<img src=https://s.natalian.org/2024-11-26/configure.png>

**Comment by m26dvd** [comment]
If the scaper associated with your local authority requires a Selenium URL, you will need to host this somewhere, either local to your Home Assistant or remotely on another device on your network. There are a number of different ways to do this depending on your setup which have been partly documented here - https://github.com/robbrad/UKBinCollectionData?tab=readme-ov-file#selenium

Hope this helps!

**Reply by kaihendry** [reply]
I'm totally confused whether Cornwall Council requires Selenium or where to configure it in Home Assistant. 😬

**Reply by m26dvd** [reply]
> I'm totally confused whether Cornwall Council requires Selenium or where to configure it in Home Assistant. 😬

Cornwall doesn’t require Selenium. 

If you “Add Service” within the integration, it should guide you through what it requires you to input for your property

**Reply by kaihendry** [reply]
It says "list index out of range". Not sure how to see the logs.
